examples/python/geometry/point_cloud_outlier_removal_radius.py

```diff
@@ -47,10 +38,10 @@ def display_inlier_outlier(cloud, ind):
 
 
 if __name__ == "__main__":
+    ptcloud_data = o3d.data.DemoICPPointClouds()
 
     print("Load a ply point cloud, print it, and render it")
-    pcd_path = os.path.join(test_data_path, 'ICP', 'cloud_bin_2.pcd')
-    pcd = o3d.io.read_point_cloud(pcd_path)
+    pcd = o3d.io.read_point_cloud(ptcloud_data.paths[2])
```

Comment:
since this example does not require fragments like ICP demo, using `SamplePointCloudPCD` might be better I think.

Reply:
Used SamplePointCloudPLY here.